examples/basics/parameters/get-tabledata.js

```diff
@@ -0,0 +1,70 @@
+const createSession = require('../../session');
+
+// inline script with a simple table
+const script = `
+exampletable:
+load * Inline [
+dim, val
+foo, 23
+bar, 42
+c, 54
+d, 21
+];
+`;
+
+const session = createSession();
+
+/**
+ * Write the tabelContent to console.
+ * @param {*} tabelContent
+ */
+function print(tableContent) {
+  for (let i = 0; i < tableContent.length; i += 1) {
+    console.log(`dim: ${tableContent[i].qValue[0].qText} val: ${tableContent[i].qValue[1].qText}`);
+  }
+}
+
+// local variable to store the generated sessionapp
+let sessionApp;
+
+session.open()
+  .then(global => global.createSessionApp())
+  .then((app) => {
+    sessionApp = app;
+    return app.setScript(script);
+  })
+  .then(() => sessionApp.doReload())
+  .then(() =>
+    /**
+     * the getTableData functions receives the data from a specified Table.
```

Comment:
the -> The
functions -> function (or possibly method)
receives -> returns
Table -> table